.github/workflows/README.md

```diff
@@ -7,22 +7,18 @@ The following workflows can be found here:
 
 ## Helm Release
 
-This workflow will lint all charts, bump the project version according to the `.bumpversion.cfg` file, create releases
-for all changed charts and provide an `index.yaml` for all packaged charts as a GitHub web page.
+This workflow will lint all charts, bump the project version according to the `.bumpversion.cfg` file, create an artifact and provide an updated `index.yaml` for all packaged charts on GitHub pages.
 
 ### Prerequisites
 
-Your helm charts need to be located inside the `charts` folder of your repository to use this workflow and you need
-a `.bumpversion.cfg` file in your root directory. Make sure to set the correct path for the `Chart.yaml` file. A minimal
-configuration could look like this:
+Your Helm charts and `.bumpversion.cfg` need to be located inside the `charts-dir` folder of your repository (repository root by default) to use this workflow. A minimal
+configuration with `charts-dir=charts` could look like this:
 
 ```cfg
 [bumpversion]
 current_version = 0.0.1
-commit = True
-tag = False
 
-[bumpversion:file:charts/my-chart/Chart.yaml]
+[bumpversion:file:charts/Chart.yaml]
```

Comment:
The description above indicates that the workflow will work for multiple charts, but with this change only a single chart is supported in the `charts` directory. How would it work for multiple?

Reply:
sorry, it doesn't work for multiple. Let me update the docs once more

Reply:
This should then also be resolved as an issue